src/ol/interaction/Draw.js

```diff
@@ -1136,7 +760,7 @@ class Draw extends PointerInteraction {
     if (targets.length) {
       this.traceState_ = {
         active: true,
-        startPx: event.pixel.slice(),
+        startCoord: event.coordinate.slice(),
```

Comment:
This change was made due to a new requirement in the Modify interaction, i.e. to not lose the trace start when dragging - a pixel would not be correct after dragging any more.